ocaml/xapi-storage/python/xapi/__init__.py

```diff
@@ -31,6 +31,13 @@
 import json
 import argparse
 
+# pylint: disable=invalid-name,redefined-builtin,undefined-variable
+# pyright: reportUndefinedVariable=false
+if sys.version_info[0] > 2:
+    long = int
+    unicode = str
+    str = bytes
+
```

Comment:
This looks like cheating. Do we strictly need this? I mean, does the real change to Python 3 take too much effort?

Reply:
Yes it feels like cheating, but the real issue is continue to support python2.
A better option would be to have the main code be python3 and have a compat hack for `sys.version_info[0] < 3`... but I just followed the current practice here (only recycled bits included in this hunk :wink:).

Reply:
It is dangerous to redefine the builtin functions, 
You will get very confused when you read the using code without refer here. (and others may even not know).